multiplexer/abci/multiplexer.go

```diff
@@ -403,7 +403,7 @@ func openDB(rootDir string, backendType db.BackendType) (db.DB, error) {
 
 func openTraceWriter(traceWriterFile string) (w io.WriteCloser, err error) {
 	if traceWriterFile == "" {
-		return
+		return w, err
```

Comment:
[flup] this will return a nil err but I think it may be more helpful if it returned something like

```suggestion
		return w, fmt.Errorf("can not open trace writer for empty file")
```